master/docs/manual/cfg-www.rst

```diff
@@ -119,10 +119,10 @@ The available classes are described here:
 
     ::
 
-        from buildbot.plugins import util
+        from buildbot.www import auth
         c['www'] = {
             # ...
-            'auth': util.UserPasswordAuth({"homer": "doh!"}),
+            'auth': auth.UserPasswordAuth({"homer": "doh!"}),
         }
 
 .. py:class:: buildbot.www.auth.HTPasswdAuth(passwdFile)
```

Comment:
We have certain _entitites_ in Buildbot, those are exposed as `buildbot.plugins.entity`, the rest goes under `buildbot.plugins.utils`...

So in this case it should be `buildbot.plugins.utils.HTPasswdAuth` and so on...